### PR TITLE
color conversion

### PR DESCRIPTION
**Pull request by aespitia**
Adding a method to convert from ScottPlot.Color to System.Drawing.Color.

#3953

function name update to ToColor

sample from linqpad

```cs
var c = ConvertColor(ScottPlot.Color.FromHex("#CCCCCC"));

c.Dump();

static System.Drawing.Color ConvertColor(ScottPlot.Color color)
{
	return System.Drawing.Color.FromArgb(color.A, color.R, color.G, color.B);	
}
```
 result from dump
 
image

### CODE REVIEW

**Comment by kebox7** [comment]
A method to convert System.Drawing.Color to ScottPlot.Color already exists

https://github.com/ScottPlot/ScottPlot/blob/fb3b1c84af3c361bf1eb8f4286ad1d4e7038afc5/src/ScottPlot5/ScottPlot5/Primitives/Color.cs#L152

**Comment by heartacker** [comment]
is it possible to add a static extend func
```
ScottPlot.Color ToSPColor(this System.Drawing.Color color)
```

**Comment by aespitia** [comment]
updated PR to go the other way from SP Color to system.drawing.color

File: src/ScottPlot5/ScottPlot5/Primitives/Color.cs [hunk 1]
```diff
@@ -359,4 +359,9 @@ public static Color RandomHue()
         float luminosity = 0.5f;
         return Color.FromHSL(hue, saturation, luminosity);
     }
+
+    public static System.Drawing.Color ConvertColor(ScottPlot.Color color)
```
Comment (kebox7) [inline src/ScottPlot5/ScottPlot5/Primitives/Color.cs:363]: Perhaps it is better to change the method name to `ToColor`, similar to `FromColor`?

Comment (aespitia) [reply src/ScottPlot5/ScottPlot5/Primitives/Color.cs]: i like it.  updated.

**Review by cataclism — APPROVED** [review]
Aligned to FromColor.

File: src/ScottPlot5/ScottPlot5/Primitives/Color.cs (hunk 1 above)
Comment (cataclism) [reply src/ScottPlot5/ScottPlot5/Primitives/Color.cs]: Hi @kebox7! If you can solve the conversation it can later be merged ;)

**Pull request merged**